### DISCUSSION TITLE
LCD screen went blank

### DISCUSSION THREAD

**Discussion by DarrellH89** [keirf/flashfloppy#954] [Q&A] [answered]
My Gotek with a STM32F105 is still working, even though the LCD (128x64) screen is blank. I've tried a spare LCD display and checked the supply voltage. Are there any other tests I can do or does this unit just need to be replaced?

Thanks,

    Darrell

**Comment by keirf** [comment, marked as answer]
Without any usb stick inserted, try holding both buttons for a few seconds.

**Reply by DarrellH89** [reply]
Thanks! Awesome, as always.
